test/integration/helpers.bash

```diff
@@ -36,15 +36,29 @@ function swarm() {
 	"$SWARM_BINARY" "$@"
 }
 
+# Retry a command $1 times until it succeeds. Wait $2 seconds between retries.
+function retry() {
+	local attempts=$1
+	shift
+	local delay=$1
+	shift
+	local i
+
+	for ((i=0; i < attempts; i++)); do
+		run "$@"
+		if [[ "$status" -eq 0 ]] ; then
+			return 0
+		fi
+		sleep $delay
```

Comment:
for ((i=0; i < attempts; i++)); do
                sleep $delay
		run "$@"
		if [[ "$status" -eq 0 ]] ; then
			return 0
		fi
		
maybe we should change the 'sleep' position, put sleep function first. The test will be ok!